messages/se.json

```
@@ -87,6 +87,9 @@
     "office": "Kontor",
     "all": "Alla"
   },
+  "event_section": {
+    "see_all_events": "Sjå alle arrangement"
```

Kan dette stemme bedre kanskje hehe? 

```suggestion
    "see_all_events": "Se alla arrangemang"
```

---

Ja, helt sikkert! Jeg brukte apertium... Takk!